### ISSUE TITLE
Holding L mouse button on 'Special Actions 'Press/Toggle Key''

### ISSUE THREAD

**Issue by Supersonic159** [Ryochan7/DS4Windows#453]
I've bound the movement of the mouse to the left control stick in the 'controls', which has worked perfectly, However, I'm looking to bind my L mouse button to the movement of the left control stick, so when I move the mouse it clicks and holds until while moving the stick. This is to simulate a touch and drag within a 'touch enabled' environment.

Within Special Actions 'Press/Toggle Key' works perfectly for keys (ex. putting 'Z' as the key pressed results in 'Z' being typed while the stick is pushed in a direction/mouse is being moved, and stops when stick is at rest) but within the options to select a key, there is no option for L mouse button, which would quickly solve my issue.

https://puu.sh/BIfrL/a181b7adda.png

Is this possible to achieve as is or is modification of DS4Windows needed?

**Comment by Ryochan7** [comment]
I don't think there is a good way to get a binding like that to work in DS4Windows. I am not sure why that window omits the portion that displays mouse controls and only allows keyboard keys to be mapped. DS4Windows can sometimes be really limiting for anything beyond simple controller button remapping. I can take a look and see if there is any real reason why that limit is in place and possibly remove that restriction if it can be done.

**Comment by Supersonic159** [comment]
Yeah it seems like a tricky thing to bind. But if you could find something that removes the limitation or even a work around, that would be incredible. I'm trying to implement this touch and this is the only way I can think of doing it.
